### ISSUE TITLE
How to add loss function only for G_A

### ISSUE THREAD

**Issue by Iarkii** [junyanz/pytorch-CycleGAN-and-pix2pix#1068]
Hi~ I'm confused about is G_A and G_B using same loss function? If it is, why can we get two different generator.  

Besides, How can I add a loss function only for G_A, I just did not find similar question in Issues.

**Comment by junyanz** [comment]
The input and output of G_A and G_B are different. (one is horse->zebra, the other is zebra->horse). Add your loss at this [line](https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix/blob/master/models/cycle_gan_model.py#L169).

**Comment by Iarkii** [comment]
Thanks a lot!
